test/gtest/conv_trans.cpp

```diff
@@ -30,6 +30,8 @@
 #include "../conv2d.hpp"
 #include "get_handle.hpp"
 
+MIOPEN_DECLARE_ENV_VAR_BOOL(MIOPEN_TEST_ALL)
```

Comment:
These common ones should be moved to a common header files instead of repeating declarations in each test.

Reply:
@amberhassaan This change is a bugfix. Your suggestion is very useful refactoring, but I highly recommend implementing it in a dedicated PR. It might be worth opening a separate ticket.

Reply:
> These common ones should be moved to a common header files instead of repeating declarations in each test.

@amberhassaan , agreed common code should be moved to a header, but environment variable setting in cTest is test based, this is to keep the design as close as original as possible for the moment, e.g. the two tests below

#   **SKIP_UNLESS_ALL** - The test should be only run if **MIOPEN_TEST_ALL**=TRUE. Intended for long tests.

1. No SKIP_UNLESS_ALL (MIOPEN_TEST_ALL)
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/8be2998d7a1210b7bc82d778f7df0d92aa65b57d/test/CMakeLists.txt#L951C35-L951C35

2. With SKIP_UNLESS_ALL (MIOPEN_TEST_ALL)
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/8be2998d7a1210b7bc82d778f7df0d92aa65b57d/test/CMakeLists.txt#L1009

Reply:
> > These common ones should be moved to a common header files instead of repeating declarations in each test.
> 
> @amberhassaan , agreed common code should be moved to a header, but environment variable setting in cTest is test based, leaving it to CPP is to keep the design as close as original as possible for the moment, e.g. the two tests below
> 
> **SKIP_UNLESS_ALL**- The test should be only run if **MIOPEN_TEST_ALL**=TRUE. Intended for long tests.
> 
>     1. No SKIP_UNLESS_ALL (MIOPEN_TEST_ALL)
>        https://github.com/ROCmSoftwarePlatform/MIOpen/blob/8be2998d7a1210b7bc82d778f7df0d92aa65b57d/test/CMakeLists.txt#L951C35-L951C35
> 
>     2. With SKIP_UNLESS_ALL (MIOPEN_TEST_ALL)
>        https://github.com/ROCmSoftwarePlatform/MIOpen/blob/8be2998d7a1210b7bc82d778f7df0d92aa65b57d/test/CMakeLists.txt#L1009